composer.json

```diff
@@ -20,7 +20,8 @@
     "require": {
         "semsol/arc2": "v2.2.4",
         "chrisboulton/php-resque": "dev-master#98fde571db008a8b48e73022599d1d1c07d4a7b5",
-        "monolog/monolog" : "~1.13"
+        "monolog/monolog" : "~1.13",
```

Comment:
I don't think they should - libraries should have fuzzy versions whereas apps lock them down.